src/runtime/audience-activity-listener.js

```diff
@@ -84,14 +84,15 @@ export class AudienceActivityEventListener {
           const pubId = encodeURIComponent(
             this.deps_.pageConfig().getPublicationId()
           );
+          const encodedSwgUserToken = encodeURIComponent(swgUserToken);
           const audienceActivityClientLogsRequest =
             this.createLogRequest_(event);
           const url = serviceUrl(
```

Comment:
Can we use `addQueryParam` function which handles encoding instead of manually appending it?

```
const url = `/publication/${pubId}/audienceactivity`;
url = addQueryParam(url, 'sut', swgUserToken);
this.fetcher_.sendBeacon(serviceUrl(url), audienceActivityClientLogsRequest);
```

Reply:
Changed!